libraries/cbor/src/writer.rs

```diff
@@ -17,6 +17,9 @@
 use super::values::{Constants, Value};
 use alloc::vec::Vec;
 
+/// Default maximum level of nested structures supported.
+const MAX_NESTING_DEPTH: i8 = i8::MAX;
```

Comment:
Would it make sense to reuse the same const as the reader? Or in other words: is there a case where it would make sense to have these 2 constants differ?

Reply:
The code pre-#329 had both `Reader::MAX_NESTING_DEPTH` and `Writer::MAX_NESTING_DEPTH` so I was echoing that; probably makes sense to have a single constant somewhere though.

Reply:
Note that we could as well inline those values. If we believe we won't change that value in the future, then that's probably the best option because it would improve readability by removing one level of indirection. Otherwise, I agree we should try to share that constant (e.g. in `lib.rs`) such that we only need to change it in one place (I don't see any reason those constants between reader and writer should differ so we should use 2 constants only when that reason becomes clear).

Reply:
Moved to inline values and dropped the constants.